src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  const longerDiscount = 50;
+  const shortDiscount = 20;
```

Comment:
Consider renaming 'shortDiscount' to 'mediumDiscount' for better readability and consistency with the task description.